tests/targetAcquisition_geolocation_test.py

```diff
@@ -1,32 +1,23 @@
 from numpy import outer
 import pytest
 import cv2
-import time
-import datetime as datetime
 import os.path
-import csv
-import simplekml
-
-import subprocess
-import pandas as pd
 
 from modules.decklinksrc.decklinkSrcWorker_taxi import DeckLinkSRC
 from modules.mergeImageWithTelemetry.mergedData import MergedData
 from modules.targetAcquisition.targetAcquisition import TargetAcquisition
 from modules.geolocation.geolocation import Geolocation
-from modules.timestamp.timestamp import Timestamp
-from modules.mergeImageWithTelemetry.mergeImageWithTelemetry import MergeImageWithTelemetry
 
 # @pytest.fixture
 def get_image():
-    img1 = cv2.imread('tests/testImages/frame1.jpg')
+    img1 = cv2.imread('tests/testImages/pylon_test.jpg')
+    # assert img1 != None
```

Comment:
Uncomment this, better to catch it early than to catch it five functions into the model.